### PR TITLE
Feature/deploy guilds script

### PR DESCRIPTION
**Pull request by rossneilson**
Looks to be working now with both of the below commands

`yarn hardhat --network localhost deploy-dxvote-develop`
`yarn hardhat --network localhost deploy-guilds-develop --registry 0x0`

### CODE REVIEW

File: scripts/DeploymentTemplates/dxvote-develop.js [hunk 1]
```diff
@@ -381,7 +381,9 @@ task("deploy-dxvote-develop", "Deploy dxvote with develop config").setAction(
             amount: "0",
           },
         },
+      ],
 
+      guildActions: [
```
Comment (AugustoL) [inline scripts/DeploymentTemplates/dxvote-develop.js:386]: Running guildActions before or after the "dao" actions wont allow run actions in between them, and thats one of the things we want to tests in the DXDGuild.

I think we should be able to run dao and guild actions at the same time, this means having a single actions array, not actions a& guildActions

File: scripts/utils/deploy-tokens.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+require("@nomiclabs/hardhat-web3");
+const { default: BigNumber } = require("bignumber.js");
+
+export async function deployTokens(deploymentConfig, accounts) {
+  const ERC20 = await hre.artifacts.require("ERC20");
+  const ERC20SnapshotRep = await hre.artifacts.require("ERC20SnapshotRep");
+
+  let tokens = {};
+  let addresses = {};
+  // await Promise.all(
```
Comment (AugustoL) [inline scripts/utils/deploy-tokens.js:13]: commended code and console log here

File: scripts/utils/deploy-tokens.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    ) {
+      return new BigNumber(previous).plus(current.amount.toString());
+    },
+    0);
+    console.log({ totalSupply });
+    console.log(totalSupply.toString());
+
+    let newToken;
+    switch (tokenToDeploy.type) {
+      case "ERC20":
```
Comment (AugustoL) [inline scripts/utils/deploy-tokens.js:33]: lets use the ERC20 case on default.

**Review by AugustoL — CHANGES_REQUESTED** [review]

File: scripts/DeploymentTemplates/dxvote-develop.js (hunk 1 above)
Comment (rossneilson) [reply scripts/DeploymentTemplates/dxvote-develop.js:386]: Good point, just means replicating some code but can do that

File: scripts/utils/deploy-tokens.js (hunk 2 above)
Comment (rossneilson) [reply scripts/utils/deploy-tokens.js:13]: Was still debugging stuff but will remove it and comment once its all working

**Review by MiltonTulli — APPROVED** [review]

**Review by AugustoL — APPROVED** [review]

**Pull request merged**